extensions/vertx-http/deployment/src/main/java/io/quarkus/vertx/http/deployment/VertxHttpProcessor.java

```diff
@@ -158,4 +161,11 @@ ServiceStartBuildItem finalizeRouter(
 
         return new ServiceStartBuildItem("vertx-http");
     }
+
+    @BuildStep
+    NativeImageConfigBuildItem build() {
+        return NativeImageConfigBuildItem.builder()
+                .addRuntimeInitializedClass("io.vertx.ext.web.handler.sockjs.impl.XhrTransport")
```

Comment:
Why not use `RuntimeInitializedClassBuildItem` instead?

Reply:
Fixed.